src/hooks/useAuth.js

```diff
@@ -0,0 +1,12 @@
+import { useContext } from 'react';
+import { authContext } from '../context/ProvideAuth';
+
+export default function useAuth() {
+  const context = useContext(authContext);
+  console.log('!!!!', authContext);
```

Comment:
Remember that clean code is best, so you should always try and clean up unused console logs prior to pushing your code.